tools/CleanMSSQLLocalDB.cmd

```diff
@@ -0,0 +1,9 @@
+@echo off
+SqlLocalDB start
+sqlcmd -S "(localdb)\mssqllocaldb" -i "%~dp0DropAllDatabases.sql" -l 32 -r0 >"DropAll.sql"
+echo Will execute:
+type DropAll.sql
```

Comment:
Up to you whether we need these two lines or, if we need them, whether to add something separating the script content from whatever follows.

Reply:
I doubt anyone will see this output unless something goes wrong